### ISSUE TITLE
@inheritdoc fails when class C implements interface B which extends interface A

### ISSUE THREAD

**Issue by myphysicslab** [jsdoc3/jsdoc#909]
Suppose that class C implements interface B, and interface B extends interface A. Then interface B shows the documentation inherited from interface A. But class C does not. Class C shows the method, but without the proper inherited documentation from interface A.

This is similar to [issue 906](https://github.com/jsdoc3/jsdoc/issues/906) "@inheritdoc fails when class C extends class B which implements interface A" which was fixed.

In the following example: AbstractSubject implements Subject interface which extends Printable interface. The problem is the toStringShort method in AbstractSubject is lacking documentation.
The resulting documentation can be seen at:
http://www.myphysicslab.com/rmnp/test20150205/out/index.html
Below is the code for this example. Also available at 
http://www.myphysicslab.com/rmnp/test20150205/test20150205.js
Using JSDoc 3.4.0-dev (Thu, 22 Jan 2015 23:02:06 GMT)

```javascript
/** @classdesc An object that has a minimal string representation via
the `toStringShort` method.
* @interface
*/
myphysicslab.lab.util.Printable = function() {};

/** Returns a minimal string representation of this object, usually
giving just identity information like the class name and name of the
object.
@method
@return {string} a minimal string representation of this object.
*/
myphysicslab.lab.util.Printable.prototype.toStringShort;

/** @classdesc A {Subject} notifies its {Observer}s when something changes in the
{Subject}.
@interface
* @extends {myphysicslab.lab.util.Printable}
*/
myphysicslab.lab.util.Subject = function() {};

/** Returns the JavaScript symbol that refers to this {Subject} for usage in a Terminal
script.
@method
@return {string} the symbol of this {Subject}.
*/
myphysicslab.lab.util.Subject.prototype.getSymbol;

/** @classdesc A *reusable forwarding class* which implements the boilerplate code
for a {Subject}.
@constructor
@implements {myphysicslab.lab.util.Subject}
*/
myphysicslab.lab.util.AbstractSubject = function() {};

/** @inheritDoc */
myphysicslab.lab.util.AbstractSubject.prototype.getSymbol = function() {
  return "foo";
};

/** @inheritDoc */
myphysicslab.lab.util.AbstractSubject.prototype.toStringShort = function() {
  return "AbstractSubject";
};
```

**Comment by hegemonic** [comment]
Ugh. Yeah, it makes sense that the fix for #906 would cause this behavior.

This will be a tricky one to solve, and the fix may not make it into 3.3.0.

**Comment by hegemonic** [comment]
Fixed on master and releases/3.3. I found a relatively straightforward fix that introduces a minor regression (#911). There's a way to work around the regression, so introducing the regression seemed like the lesser of two evils.

**Comment by myphysicslab** [comment]
Thanks for correcting this so quickly.  This one and #906 and #907 are all working, I just retested them.  However, I found a new one which I reported as #912.  At least you are getting some elaborate test cases from me!
